python/cuml/dask/solvers/cd.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.dask.common.base import BaseEstimator
+from cuml.dask.common.base import DelayedPredictionMixin
+from cuml.dask.common.base import mnmg_import
+from cuml.dask.common.base import SyncFitMixinLinearModel
+from cuml.dask.common.comms import worker_state
+
+
+class CD(BaseEstimator,
+         SyncFitMixinLinearModel,
+         DelayedPredictionMixin):
+    """
+    Model-Parallel Multi-GPU Linear Regression Model. Single Process Multi GPU
+    supported currently
+    """
+
+    def __init__(self, verbose=False, client=None, **kwargs):
+        """
+        Initializes the linear regression class.
+
+        """
+        super(CD, self).__init__(client=client,
+                                 verbose=verbose,
+                                 **kwargs)
+        self.coef_ = None
+        self.intercept_ = None
+        self._model_fit = False
+        self._consec_call = 0
+
+    def fit(self, X, y):
+        """
+        Fit the model with X and y.
+
+        Parameters
+        ----------
+        X : Dask cuDF dataframe  or CuPy backed Dask Array (n_rows, n_features)
```

Comment:
I am a little confused about the part where elastic and lasso use the cd model and fit func but they can use input of any type but cd can only accept dask cudf or dask arrays?

Reply:
All cuml.dask algorithms only do dask cudf or dask arrays, nowhere do we accept all the amount of things we do in single gpu algorithms

Reply:
the docstrings were not updated, updated docstrings should reflect that now